lib/more_core_extensions/core_ext/string/decimal_suffix.rb

```diff
@@ -0,0 +1,19 @@
+module MoreCoreExtensions
+  module DECIMAL_SI
```

Comment:
Why capital?  Following IEC60027_2 example?  I think that's because IEC is an abbreviation, so this would be `DecimalSI`.
But module & filename should match, so `module DecimalSuffix`?